src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -16,7 +29,25 @@ public class HelpWindow extends UiPart<Stage> {
     private static final String FXML = "HelpWindow.fxml";
 
     @FXML
-    private TextArea helpContent;
+    private TextFlow helpContentFlow;
```

Comment:
Really detailed and extensive research into FXML elements is evident. Well done, Shane!